backend/plugin/db/hive/hive.go

```diff
@@ -0,0 +1,171 @@
+package hive
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"io"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/beltran/gohive"
+	"github.com/pkg/errors"
+
+	"github.com/bytebase/bytebase/backend/plugin/db"
+
+	storepb "github.com/bytebase/bytebase/proto/generated-go/store"
+	v1pb "github.com/bytebase/bytebase/proto/generated-go/v1"
+)
+
```

Comment:
We'd better add `_ db.Driver = (*driver)(nil)` to show the interface it implemented